### DISCUSSION TITLE
How to get all group members' numbers?

### DISCUSSION THREAD

**Discussion by iBobik** [AsamK/signal-cli#1286] [Q&A] [answered]
I am trying to list phone numbers of all group members, but listGroups returns null for members I am not connected with, but as an admin I have permission to see them (in the app).

```
signal-cli -o json -u +1111111111 listGroups -g xxx=
[{"id":"xxx=","name":"xxx","description":"xxx","isMember":true,"isBlocked":false,"messageExpirationTime":2419200,"members":[{"number":null,"uuid":"xxx"},{"number":"+22222222,"uuid":"xxx"},
… 
```

**Comment by AsamK** [comment, marked as answer]
You can only get their number, when they send you a direct message. For those users you can use their uuid instead of their number.
